TransactionalSession public documentation (#5813)
Particular/docs.particular.net: nservicebus/outbox/index.md

## Important design considerations

* For best performance, outbox data should be stored in the same database as business data. For more information, see [_Transaction scope_](#important-design-considerations-transaction-scope) below.

> * The outbox works only in an NServiceBus message handler. For use outside the context of a message handler, the [TransactionalSession-package](/nservicebus/transactional-session) can be used.

Reviewer: ```suggestion
* The outbox works in an NServiceBus message handler without any additional dependencies. For use outside the context of a message handler, the [TransactionalSession-package](/nservicebus/transactional-session) is required.
```
